src/v/redpanda/admin/debug_bundle.cc

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "base/type_traits.h"
+#include "debug_bundle/debug_bundle_service.h"
+#include "debug_bundle/error.h"
+#include "debug_bundle/json.h"
+#include "debug_bundle/types.h"
+#include "json/document.h"
+#include "json/types.h"
+#include "redpanda/admin/api-doc/debug_bundle.json.hh"
+#include "redpanda/admin/server.h"
+#include "reflection/type_traits.h"
+#include "ssx/sformat.h"
+#include "utils/functional.h"
+
+#include <seastar/core/sstring.hh>
+#include <seastar/json/json_elements.hh>
+
+#include <boost/lexical_cast/try_lexical_convert.hpp>
+#include <fmt/core.h>
+
+#include <charconv>
+#include <chrono>
+#include <sstream>
+
+namespace {
+
+admin_server::json_status_return_type
+make_error_body(debug_bundle::error_code ec, const ss::sstring& msg) {
+    ss::httpd::debug_bundle_json::error_body res;
+    res.error_code = static_cast<int>(ec);
+    res.message = msg;
+
+    ss::http::reply::status_type status{ss::http::reply::status_type::ok};
+    switch (ec) {
+    case debug_bundle::error_code::success:
+    case debug_bundle::error_code::debug_bundle_process_running:
```

Comment:
In the case of `POST /v1/debugbundle/`, process running returns a `409`

Reply:
This is true as well with `/ebug/bundle/file/{file}`

Reply:
Done